Source/IntegrationTests/TestFiles/LitTests/LitTest/traits/NonReferenceTraits.dfy

```diff
@@ -216,14 +216,14 @@ module UninitializedConsts {
     ghost const A: AutoInit
     ghost const B: AutoInit := 15
 
-    const g: Nonempty // error: requires initialization
+    const g: Nonempty
     const h: Nonempty := 15
     ghost const G: Nonempty
     ghost const H: Nonempty := 15
 
-    const s: PossiblyEmpty // error: requires initialization
+    const s: PossiblyEmpty
```

Comment:
That's possibly the biggest potential issue but I think we are fine, since a trait is a collection of contracts, and someone should at least extend a mocked class that extends these contracts to test them.